Components/GameBoard/Messages.jsx

```diff
@@ -27,18 +27,40 @@ class Messages extends React.Component {
             'greyjoy'
         ];
 
+        this.tokens = {
+            card: { className: 'icon-card', imageSrc: '/img/cards/cardback.jpg' },
+            cards: { className: 'icon-card', imageSrc: '/img/cards/cardback.jpg' },
+            gold: { className: 'icon-gold', imageSrc: '/img/Gold.png' }
+        };
+
         this.formatMessageText = this.formatMessageText.bind(this);
     }
 
     getMessage() {
-        var index = 0;
-        var messages = this.props.messages.map(message => {
+        let index = 0;
+        let messages = this.props.messages.map(message => {
             return <div key={ 'message' + index++ } className='message'>{ this.formatMessageText(message.message) }</div>;
         });
 
         return messages;
     }
 
+    processKeywords(message) {
+        let messages = [];
+
+        for(let token of message.split(' ')) {
+            if(this.tokens[token]) {
+                let tokenEntry = this.tokens[token];
+                messages.push(<img className={ tokenEntry.className } src={ tokenEntry.imageSrc } />);
```

Comment:
Will this generate warning messages about rendering an array without the elements having unique keys?

Reply:
Yes, yes it will!